shared/Views/Shared/Components/CourseDetails/_Apply.cshtml

```diff
@@ -33,7 +33,20 @@
       Model.Course.Provider.ProviderCode == "2LR" ||
       Model.Course.Provider.ProviderCode == "254" ||
       Model.Course.Provider.ProviderCode == "12K" ||
-      Model.Course.Provider.ProviderCode == "1OT"
+      Model.Course.Provider.ProviderCode == "1OT" ||
```

Comment:
`@using System.Linq;` 
add top of file

```csharp
var allowedProviders = { "R55", "1N1" };

allowedProviders.Any(providerCode == Model.Course.Provider.ProviderCode )
```


if it gets big and too messy maybe that